src/Filters/GroupFilter.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace CodeIgniter\Shield\Filters;
+
+use CodeIgniter\Filters\FilterInterface;
+use CodeIgniter\HTTP\RequestInterface;
+use CodeIgniter\HTTP\Response;
+use CodeIgniter\HTTP\ResponseInterface;
+
+use CodeIgniter\Shield\Exceptions\GroupException;
+
+class GroupFilter implements FilterInterface
+{
+    /**
+     * Do whatever processing this filter needs to do.
```

Comment:
Please update this docblock to be descriptive of what the function is actually doing.

Reply:
The docblock has not been updated yet.

Reply:
```php
/**
 * Group Authorization Filter.
 */
class GroupFilter implements FilterInterface
{
}
```